client/src/components/HtmlEditorField/HtmlEditorField.js

```diff
@@ -80,9 +80,16 @@ class HtmlEditorField extends TextField {
     const { value } = this.props;
 
     if (value !== prevProps.value) {
-      const event = new Event('change', { bubbles: true });
+      let event;
+      if (typeof (Event) === 'function') {
```

Comment:
So IE11 doesn't support this as an interface? Is there a polyfill we can use instead of this, or is this the best approach?

Reply:
Event is supported.
![image](https://user-images.githubusercontent.com/778003/63814141-faf88300-c983-11e9-93b9-55a65745c1e9.png)

Reply:
hahahah, omg, so @Cheddam just pointed out to me that although `Event` is supported, the _constructor_ isn't 😂 
https://developer.mozilla.org/en-US/docs/Web/API/Event#Browser_compatibility